packages/fontaine/test/index.spec.ts

```
@@ -1,301 +1,81 @@
+
+            const result = generateFallbackCSS('TestFont', { /* options */ });
+
+            expect(generateFallbackCSSInternal).toHaveBeenCalledWith('TestFont', { /* options */ });
+            expect(result).toBe(mockFallback);
+        });
+    });
+
+    describe('transformTextMetrics', () => {
+        it('should return a transformed metric value', () => {
+            // Assuming transformTextMetrics expects text and options
+            const text = 'test text';
```

ESLint

[error] Extra semicolon.

(style/semi)